packages/core/src/types.ts

```diff
@@ -79,6 +79,7 @@ export interface InitializeResult {
     tools?: { listChanged?: boolean };
     prompts?: { listChanged?: boolean };
     resources?: { listChanged?: boolean; subscribe?: boolean };
+    elicitation?: Record<string, never>;
```

Comment:
why string, never?

Reply:
`Record<string, never>` just represents an empty object but I'll make it `{}` so it's a bit clearer.